### PR TITLE
feat(shields): Add splitkb.com Aurora Sweep.

### PR DESCRIPTION
**Pull request by petejohanson**
## Board/Shield Check-list
 - [x] This board/shield is tested working on real hardware
 - [x] Definitions follow the general style of other shields/boards upstream ([Reference](https://zmk.dev/docs/development/new-shield))
 - [x] `.zmk.yml` metadata file added
 - [x] Proper Copyright + License headers added to applicable files (Generally, we stick to "The ZMK Contributors" for copyrights to help avoid churn when files get edited)
 - [x] General consistent formatting of DeviceTree files
 - [x] Keymaps do not use deprecated key defines (Check using the [upgrader tool](https://zmk.dev/docs/codes/keymap-upgrader))
 - [x] `&pro_micro` used in favor of `&pro_micro_d/a` if applicable
 - [x] If split, no name added for the right/peripheral half
 - [x] Kconfig.defconfig file correctly wraps *all* configuration in conditional on the shield symbol
 - [x] `.conf` file has optional extra features commented out
 - [x] Keyboard/PCB is part of a shipped group buy or is generally available in stock to purchase (OSH/personal projects without general availability should create a zmk-config repo instead)

### CODE REVIEW

**Comment by caksoylar** [comment]
This shield had issues reported when used with `nice_view_adapter nice_view` shields, discussion around here: https://discord.com/channels/719497620560543766/719909884769992755/1037416611201695824

My guess was the way the SSD1306 Kconfig was set up with `select` instead of `if ... default y` might be breaking something difference compared to `corne`, `lily58` etc. shields which work fine with it.

**Comment by petejohanson** [comment]
> This shield had issues reported when used with `nice_view_adapter nice_view` shields, discussion around here: https://discord.com/channels/719497620560543766/719909884769992755/1037416611201695824
> 
> My guess was the way the SSD1306 Kconfig was set up with `select` instead of `if ... default y` might be breaking something difference compared to `corne`, `lily58` etc. shields which work fine with it.

Yes, I tried something a bit different trying to be "better" and failed. I've since switched back to our normal way and pushed this.

**Comment by petejohanson** [comment]
Same user reported success on the splitkb.com discord: https://discord.com/channels/574598631399751680/574738351836626944/1037661469082398731

**Comment by caksoylar** [comment]
Great! I missed the force push.

File: app/boards/shields/splitkb_aurora_sweep/Kconfig.defconfig [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+endif # SHIELD_SPLITKB_AURORA_SWEEP_LEFT
+
+if SHIELD_SPLITKB_AURORA_SWEEP_LEFT || SHIELD_SPLITKB_AURORA_SWEEP_RIGHT
+
+config ZMK_SPLIT
+	default y
+
+config ZMK_RGB_UNDERGLOW
+	select WS2812_STRIP
```
Comment (Nicell) [inline app/boards/shields/splitkb_aurora_sweep/Kconfig.defconfig:20]: I like this, we should consider refactoring other boards/shields to do this

**Review by Nicell — APPROVED** [review]

File: app/boards/shields/splitkb_aurora_sweep/Kconfig.defconfig (hunk 1 above)
Comment (petejohanson) [reply app/boards/shields/splitkb_aurora_sweep/Kconfig.defconfig:20]: Yes! I did need to revert the same change for the display stuff, due to nice!view wanting it *not* to select ssd1306 directly.

**Pull request merged**